smartBugs.py

```diff
@@ -16,22 +16,22 @@
 from time import time, localtime, strftime
 
 
-cfg_dataset_path = os.path.abspath('config/dataset/dataset.yaml')
+cfg_dataset_path = os.path.abspath(os.path.dirname(os.path.realpath(__file__)) + '/config/dataset/dataset.yaml')
 with open(cfg_dataset_path, 'r') as ymlfile:
     try:
         cfg_dataset = yaml.safe_load(ymlfile)
     except yaml.YAMLError as exc:
         print(exc)
 
 output_folder = strftime("%Y%d%m_%H%M", localtime())
-pathlib.Path('results/logs/').mkdir(parents=True, exist_ok=True)
-logs = open('results/logs/SmartBugs_' + output_folder + '.log', 'w')
+pathlib.Path(os.path.dirname(os.path.realpath(__file__)) + '/results/logs/').mkdir(parents=True, exist_ok=True)
+logs = open(os.path.dirname(os.path.realpath(__file__)) + '/results/logs/SmartBugs_' + output_folder + '.log', 'w')
 
 
 def analyse(args):
     global logs, output_folder
 
-    (tool, file, sarif_outputs, import_path, output_version, nb_task, nb_task_done, total_execution, start_time) = args
+    (tool, file, file_path_in_repo, sarif_outputs, import_path, output_version, nb_task, nb_task_done, total_execution, start_time) = args
```

Comment:
what happens if args cannot be mapped into the tuple?

Reply:
A better solution is to have a function to parse_args and a `NamedTuple` to store the values. 

https://docs.python.org/3/library/collections.html